main.py

```diff
@@ -346,12 +357,17 @@ def program_exit(status: int):  # so we don't need to import the entire sys modu
 
                         if player["PlayerIdentity"]["Incognito"]:
                             Namecolor = colors.get_color_from_team(player["TeamID"],
-                                                            names[player["Subject"]],
-                                                            player["Subject"], Requests.puuid, agent=player["CharacterID"], party_members=partyMembersList)
+                                                                   names[player["Subject"]],
+                                                                   player["Subject"], Requests.puuid,
+                                                                   agent=player["CharacterID"],
+                                                                   party_members=partyMembersList,
+                                                                   already_seen=bool(times))
```

Comment:
I don't think using `bool()` is the right choice here, at least to me it wasn't obvious that 0 is falsy.

```suggestion
                                                                   already_seen=True if times > 0 else False)
```